doc/articles/features/custom-fonts.md

```diff
@@ -8,6 +8,16 @@ The `FontFamily` of many controls (e.g. `TextBox` or `Control`) property allows
 
 ![Character Map UWP providing font information](../Assets/features/customfonts/charactermapuwp.png)
 
+## Default text font
+
+The default text font on WinUI is Segoe UI, which is default in Uno as well. However, Segoe UI isn't available on macOS, Linux, Browsers running on macOS or Linux. Therefore, starting with Uno 5.3, we introduce the ability to change the default text font for Uno Platform to OpenSans which is served through [Uno.Fonts.OpenSans NuGet package](https://nuget.org/packages/Uno.Fonts.OpenSans).
+
+If you are using Uno.Sdk, add `OpenSans` to `UnoFeatures` to switch to OpenSans. Otherwise, add a `PackageReference` to `Uno.Fonts.OpenSans` and also set `DefaultTextFontFamily` in application initialization:
```

Comment:
```suggestion
## Default text font

The default text font on WinUI is Segoe UI, which is the default in Uno as well. However, Segoe UI isn't available on macOS, Linux, or Browsers running on macOS or Linux. Therefore, starting with Uno 5.3, we introduce the ability to change the default text font for Uno Platform to OpenSans which is served through [Uno.Fonts.OpenSans NuGet package](https://nuget.org/packages/Uno.Fonts.OpenSans).

If you are using Uno.Sdk, add `OpenSans` to `UnoFeatures` to switch to OpenSans. Otherwise, add a `PackageReference` to `Uno.Fonts.OpenSans` and also set `DefaultTextFontFamily` in application initialization:
```